### PR TITLE
Aden 2559 delay close button

### PR DESCRIPTION
**Pull request by jacekjursza**
We are giving possibility of delaying the appearance of close button for lighbox ad layer. It is configurable trough DFP and by default set to 0.

### CODE REVIEW

**Comment by nandy-andy** [comment]
Code changes look good to me. However, let's try add some unit tests for `modalOasisHandler` since it's getting more complicated.

**Comment by drets** [comment]
Shouldn't we add the `closeDelay` doc to the https://github.com/Wikia/app/blob/ADEN-2559-delay-close-button/extensions/wikia/AdEngine/js/template/modal.js#L19-L25 section? :smirk_cat:

File: extensions/wikia/AdEngine/js/template/modalOasisHandler.js [hunk 1]
```diff
@@ -22,9 +22,18 @@ define('ext.wikia.adEngine.template.modalOasisHandler', [
...
 		uiFactory.init('modal').then((function (uiModal) {
 			uiModal.createComponent(modalConfig, (function (modal) {
+				var closeBtnDelay = parseInt(closeButtonDelay, 10) * 1000 || 0;
 				this.modal = modal;
 				modal.$content.append(adContainer);
 				modal.$element.width('auto');
+
+				if (closeBtnDelay > 0) {
+					modal.$close.hide();
+					setTimeout(function(){
```
Comment (drets) [inline extensions/wikia/AdEngine/js/template/modalOasisHandler.js:32]: `function(){` -> `function () {` ?

**Comment by drets** [comment]
Unit tests LGTM :+1:

**Pull request merged**